src/node_options.cc

```diff
@@ -303,8 +303,7 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             kAllowedInEnvironment);
   AddOption("--experimental-abortcontroller", "",
             NoOp{}, kAllowedInEnvironment);
-  AddOption("--experimental-json-modules",
-            "experimental JSON interop support for the ES Module loader",
+  AddOption("--experimental-json-modules", "",
             &EnvironmentOptions::experimental_json_modules,
```

Comment:
I think the environment option can be removed as well?

Reply:
Not on Node.js v15.x, v14.x, and v12.x. It can be removed in v16.x though.